jobs/build-node-image.Jenkinsfile

```diff
@@ -74,16 +76,40 @@ lock(resource: "build-node-image") {
                                                 image_tag_staging: container_registry_staging_image_tag,
                                                 manifest_tag_staging: container_registry_staging_manifest_tag,
                                                 registry_secret_path: REGISTRY_SECRET,
-                                                secret: "id=yumrepos,src=\$(pwd)/yumrepos/${stream_info.yumrepo.file}",
+                                                secret: "id=yumrepos,src=\$(pwd)/yumrepos/${stream_info.yumrepo.files[0]}",
                                                 from: build_from,
                                                 extra_build_args: ["--security-opt label=disable", "--mount-host-ca-certs", "--force"])
             }
         }
 
-        stage("Release Manifest") {
+        stage('Build Extensions Container') {
+            withCredentials([file(credentialsId: 'oscontainer-push-registry-secret', variable: 'REGISTRY_SECRET')]) {
+                // Use the node image as from
+                def build_from = container_registry_staging_manifest
+                // The Extesions container is using two repositories for now, we need to merge them to pass it as a secret
```

Comment:
Let's just drop this comment. It lives better in the pipecfg repo where we'll have to add the `rhel-9.4.repo` file to the list.